### DISCUSSION TITLE
Customised Site Action Summary Report

### DISCUSSION THREAD

**Discussion by Martin511** [akeeba/panopticon#559] [Ideas]
I would like to be able to create a site action summary report in which I can enter the time range myself and which is then generated directly once.

**Comment by nikosdion** [comment]
Overview, Site Reports. Select your site, enter your date range, click on Search.

**Comment by Martin511** [comment]
You're absolutely right, I wasn't precise enough.
I would like to filter it, e.g. without the updates found. In other words, only actions that have actually been carried out, such as backups and installed updates.
This report should then be able to be generated and downloaded as a PDF (or provided by e-mail as a PDF file)

**Reply by nikosdion** [reply]
You cannot filter out action types. You can, however, always display as many records as you want and print to a PDF file; printing to PDF is a feature of all browsers released in the past 10+ years.
